db/migrate/20141212183216_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
```

Comment:
This should actually be an `up`, since it cannot be automatically reversed. You would then define a `down` method which does the opposite in the case of a rollback.